.github/scripts/snapshot_images.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Initialize array to store digest SHAs
+declare -a digests
+
+# Fetch all docker image names
+image_names=$(gh api \
+  -H "Accept: application/vnd.github+json" \
+  -H "-GitHub-Api-Version: 2022-11-28" \
+  -H "Authorization: Token ${GITHUB_TOKEN}" \
+  --paginate "/orgs/boozallen/packages?package_type=container" | jq -r '.[] | select((.name | startswith("aissemble")) and (.name | endswith("-chart") | not)) | .name')
```

Comment:
Q:  I don't understand the part where we are looking for "-chart".  Are we pulling the image name from the Helm chart?

Reply:
@aaron-gary we're actually doing the opposite here! piping the `endswith` command to `not` negates it.

Reply:
Ah!  I see.  Thanks for explaining